app/src/main/java/com/kickstarter/extensions/EditTextExt.kt

```diff
@@ -15,3 +16,6 @@ fun EditText.onChange(input: (String) -> Unit) {
         override fun onTextChanged(s: CharSequence?, start: Int, before: Int, count: Int) {}
     })
 }
+
+/** This is an Extension function to get the string input for EditText */
+fun EditText.string() = this.text.toString()
```

Comment:
I was wondering if we should call this `text` vs `string`. I first thought it was a shortcut for `toString` and I was like 👁 why would we want that value?

Reply:
I think text would be good I'll make the change.